books/RayTracingInOneWeekend.html

```diff
@@ -2200,34 +2315,25 @@
 <div class='together'>
 After rendering we get a similar image:
 
-  ![Image 9: Correct rendering of Lambertian spheres
+  ![Image 10: Correct rendering of Lambertian spheres
   ](../images/img-1.09-correct-lambertian.png class=pixel)
 
 It's hard to tell the difference between these two diffuse methods, given that our scene of two
 spheres is so simple, but you should be able to notice two important visual differences:
 
-  1. The shadows are less pronounced after the change
-  2. Both spheres are lighter in appearance after the change
-
-Both of these changes are due to the more uniform scattering of the light rays, fewer rays are
-scattering toward the normal. This means that for diffuse objects, they will appear _lighter_
-because more light bounces toward the camera. For the shadows, less light bounces straight-up, so
-the parts of the larger sphere directly underneath the smaller sphere are brighter.
-</div>
+  1. The shadows are more pronounced after the change
+  2. Both spheres are darker in appearance after the change
 
-
-An Alternative Diffuse Formulation
------------------------------------
-<div class='together'>
-The initial hack presented in this book lasted a long time before it was proven to be an incorrect
-approximation of ideal Lambertian diffuse. A big reason that the error persisted for so long is
-that it can be difficult to:
-
-  1. Mathematically prove that the probability distribution is incorrect
-  2. Intuitively explain why a $\cos (\phi)$ distribution is desirable (and what it would look like)
+Both of these changes are due to the less uniform scattering of the light rays, more rays are
```

Comment:
"… of the light rays — more rays are …". Alternatively, use a semi-colon instead of an em dash.